src/store/mutations.ts

```diff
@@ -258,9 +257,7 @@ export default {
   },
 
   updateAttributeInvested(state: State, payload: any) {
-    state.build.character.attributes[payload.name].invested = Number(
-      payload.value
-    );
+    state.build.character.attributes[payload.name].invested = Math.min(Number(payload.value), 190);
```

Comment:
Could you reference or create a constant for 190 here (something like MAX_SKILL_INVESTED) if one doesn't exist?

Reply:
Totally 👍 https://github.com/amoeba/accharplanner/pull/398/commits/6a2c6dda768d1db1276fbe5fa7e6dc8c94899580